bridge/src/org/sparkboard/firebase_config.cljs

```diff
@@ -0,0 +1,8 @@
+(ns org.sparkboard.firebase-config
+  (:require [org.sparkboard.js-convert :refer [->clj]]))
+
+(defonce config (atom nil))
+
+(defn set-firebase-config! [x]
```

Comment:
I assume this moved from common/config is so it's accessible to the bridge?

Reply:
yep exactly